src/app/documentation/demos/code/code-highlight-snippet.ts

```diff
@@ -6,15 +6,16 @@
 import {Component} from "@angular/core";
 
 const HTML_EXAMPLE = `
+<!-- "&#123;" left curly brace / "&#125;" right curly brace -->
 <pre>
     <code clr-code-highlight="language-css">
-        .some-component {
+        .some-component &#123;
```

Comment:
thx for consideration. and seems ngNonBindable not work in angular 4 (tested). current i know two ways: `{{ '{' }}` or `&#123;` for escape. i tried `{{ '{' }}` and not work for this case.